sdk/include/reactos/undocgdi.h

```diff
@@ -26,19 +26,24 @@ typedef struct GDI_DRAW_STREAM_TAG
     DWORD   crTransparent; // transparent color.
 } GDI_DRAW_STREAM, *PGDI_DRAW_STREAM;
 
-BOOL
-WINAPI
+EXTERN_C BOOL WINAPI
 GdiDrawStream(HDC dc, ULONG l, PGDI_DRAW_STREAM pDS);
 
-BOOL
-WINAPI
+EXTERN_C BOOL WINAPI
 GetTextExtentExPointWPri(
     HDC hdc,
-    LPCWSTR lpwsz, 
+    LPCWSTR lpwsz,
     INT cwc,
-    INT dxMax, 
-    LPINT pcCh, 
-    LPINT pdxOut, 
+    INT dxMax,
+    LPINT pcCh,
+    LPINT pdxOut,
     LPSIZE psize);
 
-#endif
\ No newline at end of file
+EXTERN_C BOOL WINAPI
+GetFontResourceInfoW(
+    _In_z_ LPCWSTR lpFileName,
+    _Inout_ DWORD *pdwBufSize,
+    _Out_writes_to_opt_(*pdwBufSize, 1) PVOID lpBuffer,
+    _In_ DWORD dwType);
+
+#endif
```

Comment:
It should not be necessary to specify "EXTERN_C" everytime.
The whole header must be made c++-compatible by putting everything enclosed inside a
```
#ifdef __cplusplus
extern "C" {
#endif

// Here are all the functions, vars, etc.

// and at the end of the file:
#ifdef __cplusplus
}
#endif
```
This is done in all PSDK headers etc. see an example here too: https://git.reactos.org/?p=reactos.git;a=blob;f=sdk/lib/conutils/screen.h;hb=HEAD

Reply:
OK.